packages/store-sync/src/postgres-decoded/cleanStrings.ts

```diff
@@ -0,0 +1,10 @@
+import { SchemaToPrimitives, ValueSchema } from "@latticexyz/protocol-parser/internal";
+
+export function cleanStrings<TSchema extends ValueSchema>(
```

Comment:
I like Ponder's naming
```suggestion
export function removeNullCharacters<TSchema extends ValueSchema>(
```